sdk/trts/linux/Makefile

```diff
@@ -87,6 +87,10 @@ $(LIBTRTS): $(OBJS) $(LIBSGX_MM)
 	echo $(ASM_SRCS)
 	$(CC) $(CFLAGS) $(TCFLAGS) -c $< -o $@
 
+# The CTD requires AVX2
+ctd.o: ctd.c
+	$(CC) -mavx2 -O3 -masm=intel $(filter-out -O2,$(CFLAGS)) $(TCFLAGS) -c $< -o $@
```

Comment:
Looks it introduces a dependency to enclaves that HW must support AVX2.

Reply:
You are correct. It is also true that all Intel processors that support SGX also support AVX2.